aiohttp/web.py

```diff
@@ -221,6 +221,22 @@ def middlewares(self):
         return self._middlewares
 
     def make_handler(self, **kwargs):
+        try:
+            debug = kwargs['debug']
```

Comment:
I prefer `debug = kwargs.pop('debug', _sentinel)`
Please use this form, it's a little shorter and faster than exception based approach.